Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/README.md

```diff
@@ -616,6 +616,11 @@ Returns all connections.
 | --- | --- | --- |
 | limit | The maximum number of connections to return. Default is 50. | Optional | 
 | offset | The offset number to begin listing connections. Default is 0. | Optional | 
+| status | Get the connections that match only to a specific status. possible states: disconnected, timeout, waiting, connected. Can be a list, for example status=timeout,waiting. Possible values are: . | Optional | 
+| ip | Get the connections that match only to a specific IP. Can be a list, for example ip=1.1.1.1,8.8.8.8. Possible values are: . | Optional | 
+| platform | Get the connections that match only to a specific platform (OS). (Windows, Linux). Can be a list, for example platform=Windows,Linux . Possible values are: . | Optional | 
```

Comment:
why do we have possible values in all descriptions here?

Reply:
not sure why the plugin added those, removed them